lib/transloadit/response/assembly.rb

```diff
@@ -48,4 +48,17 @@ def rate_limit?
   def wait_time
     self['info']['retryIn'] || 0
   end
+
+  DEFAULT_RELOAD_TRIES = 600
+
+  def reload_until_finished! options = {}
```

Comment:
🎨  Please add parens around the method arguments:

```ruby
def reload_until_finished!(options = {})
end
```

This way it's more in line with the rest of our code (that needs a bit of style cleanup anyway)


.oO(we don't have an automatic style checker, we should implement that)